examples/core/gated_gcn/train.py

```diff
@@ -98,12 +100,37 @@ def evaluate(model, device, data_loader, evaluator):
     return evaluator.eval({"y_true": y_true, "y_pred": y_pred})["rocauc"]
 
 
-def main():
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        default="cora",
+        help="Dataset name ('cora', 'citeseer', 'pubmed').",
```

Comment:
I don't think these datasets are supported in this example. please list the supported ones only.

Reply:
I have changed the default value and the help info here.